lib/adapters/sequelize.js

```diff
@@ -139,7 +139,20 @@ module.exports = Bone => {
     //   throw new Error('unimplemented');
     // }
 
-    static async destroy(options = {}) {
+    static async destroy(options = {}, runOpt = {}) {
```

Comment:
It seems sequelize does not separate options like options.hooks into another parameter. Shouldn't we stick with their api design?

Reply:
https://sequelize.org/master/class/lib/model.js~Model.html#static-method-destroy